internal/pkg/deploy/cloudformation/stack/rd_web_svc.go

```diff
@@ -63,11 +65,23 @@ func (s *RequestDrivenWebService) Template() (string, error) {
 		return "", err
 	}
 
+	bucket, urls, err := convertRDWkldCustomResources(s.customResourcesURLs)
+	if err != nil {
+		return "", err
+	}
+
+	dnsDelegationRole, dnsName := convertAppInformation(s.app)
 	content, err := s.parser.ParseRequestDrivenWebService(template.ParseRequestDrivenWebServiceInput{
 		Variables:         s.manifest.Variables,
 		Tags:              s.manifest.Tags,
 		NestedStack:       outputs,
 		EnableHealthCheck: !s.healthCheckConfig.IsEmpty(),
+
```

Comment:
tiniest nit: just checking that the extra line was intentional

Reply:
! yess It was to create a visual separation between Alias-specific field with the rest